pkg/permissions/verbs.go

```diff
@@ -18,36 +21,87 @@ const (
 	DELETE = Verb("DELETE")
 )
 
+var allVerbsOrder = []Verb{GET, POST, PUT, PATCH, DELETE}
+
 // VerbSet is a Set of Verbs
-type VerbSet []Verb
+type VerbSet map[Verb]bool
```

Comment:
Do you use the value of these maps? Else, using `map[Verb]struct{}` is a more compact representation of a set